Point-of-No-Return/Collision.h

```diff
@@ -0,0 +1,41 @@
+﻿#ifndef COLLISION
+#define COLLISION
+
+#include "Mapchip.h"
+#include "Character.h"
+
+namespace Collision
+{
+	/**
+	 * 主人公周りのブロックで当たりア判定があるかどうか探す
+	 * @param hero
+	 * @param map
+	 * @param map情報のアドレス
+	 * @return　マップチップの二次元配列の中身
+	 */
+	std::vector<Vec2> SearchBlock(const Vec2& vec, const Size& size, int** map);
+
+
+	/**
+	 * ブロックに当たっているかどうかの確認関数
+	 * @param hero
+	 * @param previous
+	 * @param vec
+	 * @param size
+	 * @param vec2
+	 */
+	void BlockCheck(Hero& hero, Vec2 previous, Vec2 vec, Size size, std::vector<Vec2> vec2);
```

Comment:
```suggestion
	void CheckBlock(Hero& hero, Vec2 previous, Vec2 vec, Size size, std::vector<Vec2> vec2);
```
ですよね？

で、「ブロックに当たっているか」を判定する関数なのであれば、戻り値はboolにすべきだし名前も変えるべきですね。

Reply:
ですね、bool 型に変更できるようにcode変更します